docs/docs/guide/introduction/adopting-tuist/migrate-from-xcodeproj.md

```diff
@@ -71,4 +71,4 @@ They allow defining your own abstractions for defining your projects, and most i
 Here's a list of community-developed tools that can aid you on the migration process:
 
 - [xcdiff](https://github.com/bloomberg/xcdiff): A tool which helps you diff xcodeproj files.
-- [Xcode Build Settings](https://xcodebuildsettings.com/): A reference for all the build settings available and their meaning.
```

Comment:
https://xcodebuildsettings.com/ ceased to exist. meanwhile Apple's provided https://developer.apple.com/documentation/xcode/build-settings-reference seems a valid alternative